test/problematic_tests/save_genome_test.py

```diff
@@ -192,19 +217,66 @@ def test_one_genome_with_hidden(self):
         ret = self.getImpl().save_one_genome(self.ctx, params)[0]
         self.check_save_one_genome_output(ret, genome_name)
 
+    def test_genomes(self):
+        self.start_test()
+        genome_name = 'test_genome'
+        inputs = [
+            {
+                'name': genome_name,
+                'data': self.test_genome_data,
+            }
+        ]
+        params = {'workspace_id': self.wsID, 'inputs': inputs}
+        ret = self.genome_interface.save_genome_mass(params, validate_genome=True)[0]
+        self.check_save_one_genome_output(ret, genome_name)
+
+    def test_genomes_with_hidden(self):
+        self.start_test()
+        genome_name = 'test_genome_hidden'
+        inputs = [
+            {
+                'name': genome_name,
+                'data': self.test_genome_data,
+                'hidden': 1,
+            }
+        ]
+        params = {'workspace_id': self.wsID, 'inputs': inputs}
+        ret = self.genome_interface.save_genome_mass(params)[0]
+        self.check_save_one_genome_output(ret, genome_name, warnings=[])
```

Comment:
This doesn't actually test that the genome is hidden

Reply:
why?

Reply:
Because there's nothing tin check_save_one_genome_output that checks the genome is hidden

Reply:
https://github.com/kbaseapps/GenomeFileUtil/pull/212#discussion_r1826117290